C/CeresSolver/build_tarballs.jl

```diff
@@ -37,17 +45,13 @@ products = Product[
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    # CeresSolver is removing OpenMP and dependencies for OpenMP are dropped
-    # https://github.com/ceres-solver/ceres-solver/issues/886
-    # Eigen_jll v0.3.4 throws errors on powerpc64le with older GCC versions
     BuildDependency("Eigen_jll"),
     Dependency("glog_jll"),
-    # Metis replaces SuiteSparse on Windows
     Dependency("METIS_jll"),
-    Dependency("OpenBLAS32_jll"),
-    # Hard code the version now as the latest v7.0.1 does not get recognized
-    Dependency("SuiteSparse_jll", v"5.10.1")
+    Dependency("libblastrampoline_jll"; compat="5.8.0"),
+    Dependency("SuiteSparse_jll"; compat="7.2.1"),
```

Comment:
I think that based on https://github.com/JuliaPackaging/Yggdrasil/issues/8465#issuecomment-2044682817 this should be
```suggestion
    Dependency("SuiteSparse_jll"; compat="~7.2.1"),
```
?